pnpm-lock.yaml

```diff
@@ -7584,6 +7584,9 @@ packages:
   classnames@2.2.6:
     resolution: {integrity: sha512-JR/iSQOSt+LQIWwrwEzJ9uk0xfN3mTVYMwt1Ir5mUcSN6pU+V4zQFFaJsclJbPuAUQH+yfWef6tm7l1quW3C8Q==}
```

Comment:
I would strongly suggest installing classnames 2.5.1 in @plone/volto-slate as well in order to dedupe the dependency

Reply:
Done.